modules/utxo-lib/src/bitgo/dash/DashTransaction.ts

```diff
@@ -40,8 +40,8 @@ export class DashTransaction extends UtxoTransaction {
     return new DashTransaction(tx.network, tx);
   }
 
-  static fromBuffer(buffer: Buffer, noStrict: boolean, network: Network): DashTransaction {
-    const baseTx = UtxoTransaction.fromBuffer(buffer, true, network);
+  static fromBufferDash(buffer: Buffer, noStrict: boolean, network: Network): DashTransaction {
```

Comment:
Method removal. Please record as BREAKING CHANGE in commits.

Reply:
Looks like this is a problem with trying to have `Dash` and `Zcash` transaction only inherit `UtxoTransaction<number>`, implementing `TNumber` for them instead (see https://github.com/BitGo/BitGoJS/pull/2533#discussion_r928551322)

Reply:
We'll still need it recorded as a breaking change or refactor it.

Reply:
Yep, refactoring